namespaces/istio-system/gateway.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-system-ingress-gateway
+  namespace: 'istio-system'
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 80
+        name: http
+        protocol: HTTP
+      hosts:
```

Comment:
This host is wildcard `*` as all traffic into the cluster should be HTTPS and therefore be redirected.